src/client/components/forms/userCollection.js

```diff
@@ -161,8 +161,14 @@ class UserCollectionForm extends React.Component {
 		}));
 		const initialName = this.state.collection.name;
 		const initialDescription = this.state.collection.description;
-		const initialPrivacy = this.state.collection.public ? 'Public' : 'Private';
 		const initialType = this.state.collection.entityType;
+		let initialPrivacy;
+		if (this.props.collection.name) {
+			initialPrivacy = this.state.collection.public ? 'Public' : 'Private';
+		}
```

Comment:
What does the new condition do?
Is it to check if the collection already exists (vs a new collection)?

Reply:
Yes. If we're editing the collection, then it should show the proper privacy settings and if we're creating a new collection, it should show 'Public'. This is what we want to achieve right?